hazelcast/src/test/java/com/hazelcast/map/EntryProcessorBouncingNodesTest.java

```diff
@@ -109,6 +109,7 @@ public void testEntryProcessorWhileTwoNodesAreBouncing() {
         HazelcastInstance steadyMember = bounceMemberRule.getSteadyMember();
         final IMap<Integer, ListHolder> map = steadyMember.getMap(MAP_NAME);
         while (iteration < ITERATIONS) {
+            System.out.println(String.format("Iteration %d of %d", iteration + 1, ITERATIONS));
```

Comment:
minor: Would you mind adding a logger? Maybe it could be useful for you if these logs were to be ordered with hz logs also.